File: C++/iniciante/1040.cpp
#include <iostream>
#include <iomanip>

using namespace std;

int main() {

    double n1,n2,n3,n4,m,ne;
    cin>>n1>>n2>>n3>>n4;

    cout<<fixed<<setprecision(1);
    m = (n1*2+n2*3+n3*4+n4)/10;
    if(m>=7.0){
        cout<<"Media: "<<m<<endl;
        cout<<"Aluno aprovado."<<endl;}
    else if(m<5.0){
        cout<<"Media: "<<m<<endl;
        cout<<"Aluno reprovado."<<endl; }
    else{
        cout<<"Media: "<<m<<endl;
        cout<<"Aluno em exame."<<endl;
        cin>>ne; //nota do exame
        m = (m+ne)/2;
        if(m>=5.0){
            cout<<"Nota do exame: "<<ne<<endl;
            cout<<"Aluno aprovado."<<endl;
            cout<<"Media final: "<<m<<endl;}
        else if(m<5.0){
            cout<<"Nota do exame: "<<ne<<endl;
            cout<<"Aluno reprovado."<<endl;}
}

    return 0;
}

File: python 3/iniciante/1040.py
#1040 - Média 3
n1,n2,n3,n4 = map(float,input().split())
media = (n1*2+n2*3+n3*4+n4*1)/10
print('Media: {:.1f}'.format(media))
if media>=7.0:
    print('Aluno aprovado.')
elif media>=5.0:
    print('Aluno em exame.')
    exame = float(input('Nota do exame: '))
    media = (media+exame)/2
    if media>=5.0:
        print('Aluno aprovado.')
    else:
        print('Aluno reprovado.')
    print('Media final : {:.1f}'.format(media))
else:
    print('Aluno reprovado.')
